utils/wasm-builder/src/wasm_project.rs

```diff
@@ -456,6 +460,7 @@ extern "C" fn metahash() {{
         fs::create_dir_all(&self.target_dir).context("Failed to create target directory")?;
 
         let mut wasm_files = vec![(original_wasm_path.clone(), file_base_name.clone())];
+        self.generate_bin_path(file_base_name)?;
```

Comment:
for better readability since the `wasm_files` vec is created in loop
```suggestion
        self.generate_bin_path(file_base_name)?;

        let mut wasm_files = vec![(original_wasm_path.clone(), file_base_name.clone())];
```